distribution/lib/Standard/Base/0.0.0-dev/src/Data/Range.enso

```diff
@@ -200,6 +200,22 @@ type Range
                     @Tail_Call go current+self.step
             go self.start
 
+    ## PRIVATE
+       ADVANCED
+
+       Applies a function for each element in the range. Exits early if the body
+       produces an `Error`.
+    each_propagate : (Integer -> Any) -> Nothing ! Error
```

Comment:
```suggestion
       Executes a function for each element in the range. Exits early if the body
       produces an `Error`.
    each_propagate : (Integer -> Nothing) -> Nothing ! Error
```

nitpick: but IMO `apply` is more fit for a map-like pure operations where the result is somehow retained, when we are talking about functions with side-effects, IMO `execute` fits better. Feel free to ignore though.